source/leuven/slurm-accounting.rst

```diff
@@ -3,58 +3,81 @@
 Slurm Accounting
 ========================
 
-The :ref:`wICE <wice_t2_leuven>` cluster uses the Slurm Account Manager (SAM) for credit accounting. We developed so called SAM-commands to mimic the behavior of previously used MAM-commands (from the MOAB account manager). From the user point of view it should mostly be enough to change the prefix ``mam`` in the accounting command into ``sam``.
-
-The accounting system is very similar to a regular bank. Individual users have accounts that will be charged for the jobs they run. However, the number of credits on such accounts is fairly small, so
-research projects will typically have one or more project accounts associated with them. Users that are project members can have their project-related jobs charged to such a project account. In this how-to,
-the technical aspects of accounting are explained.
+The :ref:`wICE <wice_t2_leuven>` cluster uses the Slurm Account Manager (SAM) 
+for credit accounting.
+The reason is that having a valid Slurm account with sufficient credits in it
+is mandatory for running jobs on KU Leuven Tier-2 machines. 
+We developed so called SAM-commands to mimic the behavior of previously used 
+MAM-commands (from the MOAB account manager). 
+From the user point of view it should mostly be enough to change the prefix ``mam`` 
+in the accounting command into ``sam``.
+
+The accounting system is very similar to a regular bank. 
+Individual users have accounts that will be charged for the jobs they run. 
+However, the number of credits on such accounts is fairly small, so research projects 
+will typically have one or more project accounts associated with them. 
```

Comment:
I am not entirely sure what you mean here. Perhaps:

The accounting system is somewhat similar to a regular bank. A research group typically has one or more credit accounts, for instance a separate credit account for each project. When credits are purchased, they are depositied in a credit account. All users that are a member of such a credit account can withdraw credits from it by running jobs.
A users can also request introduction credits, in that case a credit account will be opened specifically for this user with a limited amount of free credits.

Reply:
Thanks for providing an improved text.
fixed via `d24f13d`.